### ISSUE TITLE
Issue with requirements.txt

### ISSUE THREAD

**Issue by marijaselakovic** [crate/cratedb-examples#302]
When installing requirements.txt in` cratedb-vectorstore-rag-openai-sql.ipynb` notebook in Google Colab I got the following ResolutionImpossible Error:

```
ERROR: Cannot install -r https://raw.githubusercontent.com/crate/cratedb-examples/main/topic/machine-learning/llm-langchain/requirements.txt (line 4), crate[sqlalchemy]>=0.34 and langchain[cratedb,openai]==0.1.4 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested crate[sqlalchemy]>=0.34
    cratedb-toolkit 0.0.5 depends on crate[sqlalchemy]>=0.34
    langchain[cratedb,openai] 0.1.4 depends on crate[sqlalchemy]<0.35.0 and >=0.34.0; extra == "cratedb"

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
```
@amotl is it possible for you to check this and loose the range of packages?

**Comment by amotl** [comment]
That command seems to work on my machine, I've tested it with two different virtualenvs using Python 3.11 and Python 3.12.
```
pip install -r https://github.com/crate/cratedb-examples/raw/main/topic/machine-learning/llm-langchain/requirements.txt
```

**Comment by marijaselakovic** [comment]
Can you please test it in Google Colab? Everything worked fine on my machine as well.

**Comment by amotl** [comment]
Fixed with #312, thank you very much.
